proto/proxy/destination/destination.proto

```diff
@@ -78,8 +78,24 @@ message WeightedAddr {
   common.TcpAddress addr = 1;
   uint32 weight = 3;
   map<string, string> metric_labels = 4;
+  TlsVerification tls_verification = 5;
```

Comment:
I suggest we call this `TlsIdentity tls_identity = 5;`

Reply:
Thanks, I was having a hard time naming this!